server/vcs/provider/github/converter/comment_event_test.go

```diff
@@ -63,9 +97,110 @@ func TestCommentEvent_Convert(t *testing.T) {
 			Hostname: "github.com",
 			Type:     models.Github,
 		},
-	}, commentEvent.BaseRepo)
+	}
+
+	modelPull := models.PullRequest{
+		Num:        *pull.Number,
+		HeadCommit: *pull.Head.SHA,
+		URL:        *pull.HTMLURL,
+		HeadBranch: *pull.Head.Ref,
+		BaseBranch: *pull.Base.Ref,
+		Author:     *pull.User.Login,
+		State:      models.OpenPullState,
+		BaseRepo:   modelRepo,
+	}
+
+	When(githubPullGetter.GetPullRequest(modelRepo, *comment.Issue.Number)).ThenReturn(&pull, nil)
+	commentEvent, err := subject.Convert(&comment)
+	Ok(t, err)
+	Equals(t, modelRepo, commentEvent.BaseRepo)
```

Comment:
for new code, i wanna stop using these helpers, we should just use:

`assert.NoError()` and `assert.Equal()`